apps/consent-das/src/services/create.ts

```diff
@@ -145,3 +146,49 @@ export const createClinicianInvite = async (
 			return failure('SYSTEM_ERROR', 'An unexpected error occurred.');
 		});
 };
+
+type CreateSignatureFailureStatus = SystemError;
+/**
+ * Creates a Signature entry in the Consent DB
+ * @param participantId Participant ID (optional)
+ * @param signature Signature data
+ * @param authenticationMethod Authentication method used to create the signature
+ * @returns Signature object from Consent DB
+ */
+export const createSignature = async ({
+	participantId,
+	signature,
+	authenticationMethod,
+	createdAt,
+}: {
+	participantId?: string;
```

Comment:
Let's make `participantId` required here and update the JSDoc comment above 🙂 
```suggestion
	participantId: string;
```